charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 3.51.2
+version: 3.51.3
```

Comment:
since you are updating a default value `datadog.securityAgent.runtime.securityProfile.enabled` could you bump the monior version of the chart? 🙇